pages/plugins/writing.md.erb

```diff
@@ -19,14 +19,17 @@ steps:
 
 ## Step 1: Create a new git repository
 
-Every Buildkite plugin is a Git repository, ending in `-buildkite-plugin`. This suffix is required: Buildkite automatically adds `-buildkite-plugin` to the plugin name specified in the `pipeline.yml`. Let's create a new Git repository following these naming conventions:
+The most common kind of Buildkite plugin is a Git repository, with a descriptive name ending in `-buildkite-plugin`. This suffix is required to allow using the `user/plugin-name` syntax in pipelines. Let's create a new Git repository following these naming conventions:
 
 ```shell
 mkdir file-counter-buildkite-plugin
 cd file-counter-buildkite-plugin
 git init
 ```
 
+> 🚧 The `-buildkite-plugin` suffix
+> Not including the `-buildkite-plugin` suffix in the repository name not only will prevent you from using the `user/plugin-name` syntax in the pipeline (you will need to reference the plugin with a full URL instead) but also makes it **more difficult for community members to find and use the plugin**. And even when you know it will be used through a full URL, only in your organization, or it is a vendored plugin the suffix should help identify the purpose of its contents to anyone looking at it. 
```

Comment:
I think this reads more clearly and breaks up the paragraph:

```suggestion
> We recommend using the `-buildkite-plugin` suffix in the repository name because:
<ul>
  <li>You can reference the plugin in pipelines using the <code>user/plugin-name</code> syntax rather than the full URL.</li>
  <li>It makes it easier for community members to find and use the plugin if you make it public.</li>
  <li>It communicates the purpose of the code.</li>
</ul>
```

Reply:
I didn't quite like the mix of HTML... but could not find a work around it :(